src/main/xsl/docs.xsl

```diff
@@ -109,30 +109,26 @@
                     <xsl:text>] </xsl:text>
                     <xsl:value-of select="created"/>
                 </small>
-                <small>
-                    <a href="{links/link[@rel='short']/@href}" style="display:block">
-                        link
-                    </a>
-                </small>
             </td>
             <td>
                 <a href="{links/link[@rel='delete']/@href}">delete</a>
-            </td>
-            <td>
-                <xsl:apply-templates select="friends"/>
+                <a href="{links/link[@rel='short']/@href}" style="display:block">
+                    share
+                </a>
             </td>
             <td>
                 <xsl:apply-templates select="visibility"/>
             </td>
         </tr>
     </xsl:template>
-    <xsl:template match="friends">
+<!--    <xsl:template match="friends">
```

Comment:
@maurezen I did it as requested in the original ticket
perhaps you're right...
I'll remove it